cpp/src/com/google/idea/blaze/cpp/BlazeCWorkspace.java

```diff
@@ -182,6 +186,28 @@ private static CidrCompilerSwitches buildSwitchBuilder(
     return combinedBuilder.build();
   }
 
+  private static void logResolveConfiguration(BlazeResolveConfiguration config) {
+    final var builder = new StringBuilder();
```

Comment:
You do some computations here which are ignored by default (because log level is not set), so it's better to wrap them with `if(logger.isTraceEnabled())`